ExamenPGG.Business/Squares/Types/Ladder.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ExamenPGG.Business.PlayerObject;
+
+namespace ExamenPGG.Business.Squares.Types
+{
+    public class Ladder : BaseSquare, ISquare
```

Comment:
i don't think it's necessary to implement the ISquare interface if you already did this in the baseSquare class.